db/schema.rb

```diff
@@ -72,8 +72,8 @@
     t.datetime "created_at",                           null: false
     t.datetime "updated_at",                           null: false
     t.string   "email"
-    t.string   "github_login"
     t.string   "credit_card_form_url"
+    t.string   "github_login"
```

Comment:
yeah that keeps happening to me. I try to stash instead of commit those schema changes, but sometimes I miss them in large commits :/